docs/yaml-preprocessing.md

```diff
@@ -456,6 +456,7 @@ things: !$escape
 ```
 
 * `!$string` convert data to a YAML string
+`!$toYaml` is an alias of `!$string`
```

Comment:
How about `toYamlString`?

Reply:
We should also documented the alternate `{{toyaml xxx}}` version and `{{tojson xxx}}', `{{tojsonPretty xxx}}`. Speaking of them, should the names of each version have the same casing?

Reply:
> How about `toYamlString`?

Sure 👍 

> We should also document the alternate `{{toyaml xxx}}` version and `{{tojson xxx}}`, `{{tojsonPretty xxx}}`. Speaking of them, should the names of each version have the same casing?

Those were documented in #251. 

I agree they should be consistent – it seem like adding `camelCase` aliases to those that don't match the format (in another PR) would be the best way forward – what do you think?

Reply:
👍

Reply:
Done in #253.